src/internal/pachd/pachw.go

```diff
@@ -59,6 +85,7 @@ func (pachwb *pachwBuilder) buildAndRun(ctx context.Context) error {
 		pachwb.initInternalServer,
 		pachwb.registerAuthServer,
 		pachwb.registerPFSServer, //PFS seems to need a non-nil auth server.
+		pachwb.registerEnterpriseServer,
```

Comment:
The other pachs register enterprise before auth and I think we should do that here.  auth also requires an identity server, but I guess we don't call methods that actually dereference the identity server?  dunno.